docs/sources/alerting/unified-alerting/grafana-managed-number-alerts.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Numeric data Grafana manged alert rules"
+description = "Number data based Grafana manged alert rules"
+keywords = ["grafana", "alerting", "guide", "rules", "create"]
+weight = 400
++++
+
+# Alerting on numeric data
+
+With certain data sources number data that is not time series can be directly alerted on, or passed into Server Side Expressions (SSE). This allows for more efficiency by doing more processing within the data source, and can also simplify alert rules.
+
```

Comment:
```suggestion
Among certain data sources numeric data that is not time series can be directly alerted on, or passed into Server Side Expressions (SSE). This allows for more processing and resulting efficiency within the data source, and it can also simplify alert rules.
```